### ISSUE TITLE
I can't find many files

### ISSUE THREAD

**Issue by SunshineXiang** [ReedOnePeck/MindDiffuser#3]
Hello, Sir:
In the main function, many parameters are missing and cannot be run.
For example, in the Image reconstruction file and def main():
1.Semantic_feature_extraction.py 
I can't find LDM_model
1.model_typology = pd.read_csv( '/nfs/diskstation/DataStation/ChangdeDu/luyizhuo/model_typology.csv')
I can't find model_typology.csv
2.n_feats
decoding_target_layers(args.CLIP_feature_decoder_path, args.mask_path, args.val_CLIP_mean_path,
							 args.val_CLIP_std_path, args.target_layers, 25, 'fastl2',n_feats, args.recons_img_idx)
3.loss_CLIP_weight
loss = MSE_CLIP_mask(masks=feature_masks, target=CLIP_target, generated=CLIP_generated,
                             weight=loss_CLIP_weight)
4.In the Feature extractor files 
    parser = argparse.ArgumentParser(description='CLIP text feature extraction')
    parser.add_argument('--stable_diffusion_config_path',  default='', type=str)
    parser.add_argument('--stable_diffusion_skpt_path',  default='', type=str)
    parser.add_argument('--coco_ids_path',  default='', type=str)
    parser.add_argument('--stimuli_data_path',  default='', type=str)
    parser.add_argument('--feature_saved_path',  default='', type=str)
    args = parser.parse_args()
Where are the paths to these files?
5.I can't find coco_ids_path and  stimuli_data_path
Semantic_feature_extraction.py
I can't find LDM_model.
Can you help me  and Can you give me some fully running code ？
Thanks

**Comment by ReedOnePeck** [comment]
Sure, you can find ways to run the project quickly here： 
https://github.com/ReedOnePeck/MindDiffuser/issues/2#issue-1874693977
If you have any other questions, please feel free to communicate。

**Comment by ReedOnePeck** [comment]
2.As for args.target_layers you can find it in the body of the paper, or you can set the layers you want yourself
2. you can set “args.recons_img_idx” you want yourself
4. parser.add_argument('--stimuli_data_path', default='', type=str)
parser.add_argument('--feature_saved_path', default='', type=str)
These two are custom parameters.

**Comment by SunshineXiang** [comment]
I am not sure how to set this up and download



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ReedOnePeck/MindDiffuser"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月30日(星期六) 晚上6:42
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [ReedOnePeck/MindDiffuser] I can't find many files (Issue #3)





 
Sure, you can find ways to run the project quickly here：
 #2 (comment)
 If you have any other questions, please feel free to communicate。

**Comment by ReedOnePeck** [comment]
Which step in this do you not know how to set up. Please be more specific.

**Comment by SunshineXiang** [comment]
In the Semantic_feature_extraction.py file,&nbsp;
&nbsp; &nbsp; &nbsp;parser.add_argument('--coco_ids_path',&nbsp; default='./', type=str)
I don't know how to make coco ids path,
I am looking forward to your reply,
Thanks!



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ReedOnePeck/MindDiffuser"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月30日(星期六) 晚上9:12
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [ReedOnePeck/MindDiffuser] I can't find many files (Issue #3)





 
Which step in this do you not know how to set up. Please be more specific.

**Comment by ReedOnePeck** [comment]
You can refer to this code  https://github.com/styvesg/nsd

**Comment by SunshineXiang** [comment]
Thank you very much for your guidance！
&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ReedOnePeck/MindDiffuser"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月30日(星期六) 晚上10:03
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [ReedOnePeck/MindDiffuser] I can't find many files (Issue #3)





 
You can refer to this code  https://github.com/styvesg/nsd

**Comment by SunshineXiang** [comment]
1.In the detail_extracttion.py 
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;'--stim_saved_path', I give 73000 images
2.In the Structural_feature_extraction.py
&nbsp; &nbsp;'--stim_sub1_trn', Why define 8859 images when I don't see where it is defined.
3.The data I read out was 10,000 images per subject.
&nbsp;(10000, 3, 425, 425)
I am looking forward to your reply.&nbsp;
Thanks!&nbsp;




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "ReedOnePeck/MindDiffuser"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月7日(星期六) 凌晨0:20
***@***.***&gt;;
***@***.***&gt;;"State ***@***.***&gt;;
主题:&nbsp;Re: [ReedOnePeck/MindDiffuser] I can't find many files (Issue #3)





 In the training dataset of NSD, there are 8859 images. To facilitate access, I used the "Image" tool to convert each image into a size of 512x512, and saved them as a .npy file with dimensions 8859x3x512x512. 
 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人:                                                                                                                        "ReedOnePeck/MindDiffuser"                                                                                    ***@***.***&amp;gt;; 
 发送时间:&amp;nbsp;2023年10月6日(星期五) 晚上8:16 
 ***@***.***&amp;gt;; 
 抄送:&amp;nbsp;"Yizhuo ***@***.***&amp;gt;;"State ***@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [ReedOnePeck/MindDiffuser] I can't find many files (Issue #3) 
 
 
 
 
 
   
 Hello Sir, 
  I need your help, 
  In the detail_extracttion.py 
  --stim_saved_path, I don't know how to make this file 
  I am looking forward to your reply. 
  Thanks!
